lib/ansible/modules/windows/win_domain_controller.py

```diff
@@ -48,6 +48,12 @@
   local_admin_password:
     description:
       - password to be assigned to the local C(Administrator) user (required when C(state) is C(member_server))
+  read_only:
```

Comment:
missing version_added, also from examples (nice to add in the 'name' field)

Reply:
changes added